corehq/messaging/scheduling/models/abstract.py

```diff
@@ -443,6 +443,12 @@ def get_translation_from_message_dict(self, domain_obj, message_dict, preferred_
         :param message_dict: a dictionary of {language code: message}
         :param preferred_language_code: the language code of the user's preferred language
         """
+
+        # If: message_dict contains no translations i.e. only contains * ;
+        # Then: treat it as an alert and send the message.
```

Comment:
these are all alerts only. We can just say
`return untranslated content, if no translations set`